### PR TITLE
add workaround to fix loading on mobile platforms (fixes #57)

### LINKED ISSUE #57: Fix loading on mobile (Android + iOS for Gear VR, Daydream, Cardboard)
@caseyyee and I figured out a simple hack to add to the `index.html` before calling `UnityLoader.instantiate(…)`:

```js
UnityLoader.SystemInfo.mobile = false;
```

@caseyyee: We ought to figure out where and why in the Unity Loader there's a check to disable loading on mobile. Does @juj have ideas about why/where this code is?

### PR DESCRIPTION
**Pull request by cvan**
r? @caseyyee
cc @juj

### CODE REVIEW

File: Assets/WebGLTemplates/WebVR/index.html [hunk 1]
```diff
@@ -11,26 +11,35 @@
...
   <script src="%UNITY_WEBGL_LOADER_URL%"></script>
   <script>
-    var gameInstance = UnityLoader.instantiate("gameContainer", "%UNITY_WEBGL_BUILD_URL%", {
-      // Preserve drawing buffer is needed for WebVR content to be mirrored out into canvas.
-      Module: {"webglContextAttributes": { "preserveDrawingBuffer" : true }},
-      onProgress: unityProgress
-    });
+    /* global UnityLoader */
+    (function () {
+      UnityLoader.SystemInfo.mobile = false;  // Workaround to force UnityLoader to actually load on mobile.
```
Comment (cvan) [inline Assets/WebGLTemplates/WebVR/index.html]: ❗️ this is the only relevant change. (the rest are style nits.)

**Comment by caseyyee** [comment]
@cvan have you tried passing it into the Unity loader config object?
```
SystemInfo {
    mobile: true
}
```

Or just ``` mobile: true ``` might work?

Will email Unity folks on this and cc you as well to find out.

**Comment by caseyyee** [comment]
Might be able to do this through the compatibility check callback?  see:

https://stackoverflow.com/questions/44146241/unity-mobile-webgl-compatibility

Might want to see if the performance is viable first before turning it on anyways.  Just because it can run, doesn't mean it's useable.

**Comment by cvan** [comment]
I tried both of those methods (based on Unity Forums and Stack Overflow), and it didn't work. I tried overriding `compatibilitycheck` and `compatibilityCheck` and on `Module` too. None of those methods worked.

One of them does, but I couldn't figure out the code path. I'm sure there's a better workaround.

> Might want to see if the performance is viable first before turning it on anyways. Just because it can run, doesn't mean it's useable.

That's certainly true. We could add a 2D/VR dialog for when latency gets (too) bad.

The example demo is runnable on my Samsung Galaxy S8. It loads on my iPhone 7, albeit a bit unusably. I can add a compatibility table to the README. That'd probably be helpful regardless of this change.

**Comment by cvan** [comment]
@caseyyee: oof, strange. when you say "remove this," do you mean revert this or merge my original change? because it looks like the change did get merged in f4f467f2b049e039683592a1e55e23e40fc38336 (although I'm not sure why there are two prior merge commits, but whatever).

I take it that this is fine and that you did intend to merge this change, yeah?

**Comment by caseyyee** [comment]
I was just referring the the duplicate commit.  No big deal really.  I intended to merge it.

The issue I think was with the github merge conflict tool and probably some user error on my part in using it.

All good!

**Pull request merged**